### PR TITLE
Fix linting issue and remove abis dir

### PR DESCRIPTION
**Pull request by sorpaas**
* Fix problems in default Migrations file that causes linting issues.

### CODE REVIEW

**Review by kirushik — APPROVED** [review]
:ship:

**Comment by sorpaas** [comment]
`constructor()` apparently breaks BuildKite analyzer, but I guess it's better to fix it in the analyzer because we consider it to be informational/optional now. Otherwise Solidity will show the deprecation warnings.

We're not testing Migrations.sol contract anyway and it won't break testing on other contracts.

**Comment by kirushik** [comment]
@sorpaas OK, I'll play with this later and submit a bugreport to the Slither team.

**Comment by sorpaas** [comment]
@kirushik I realized that running Slither locally never gives me this error -- it looks like it's fixed upstream but our Buildkite container is outdated. We'll probably just need to update that.

**Review by andresilva — APPROVED** [review]
LGTM

File: contracts/Migrations.sol [hunk 1]
```diff
@@ -3,24 +3,24 @@ pragma solidity ^0.4.17;
 
 contract Migrations {
 	address public owner;
-	uint public last_completed_migration;
+	uint public lastCompletedMigration;
```
Comment (andresilva) [inline contracts/Migrations.sol]: This probably breaks truffle's migration system since it expects a getter with this name.

**Review by andresilva — CHANGES_REQUESTED** [review]

File: contracts/Migrations.sol (hunk 1 above)
Comment (sorpaas) [reply contracts/Migrations.sol]: I think maybe we should just disable linting for Migrations.sol altogether...

**Comment by sorpaas** [comment]
Fixed the Migration.sol contract problem by:

1. Disable linting for that file.
2. Remove constructor and use direct assignment to avoid `constructor()` not supported issue.

**Review by andresilva — APPROVED** [review]

**Pull request merged**